inputFiles/poromechanics/nonlinearAcceleration/validationCase/validationCase.xml

```diff
@@ -0,0 +1,498 @@
+<Problem>
```

Comment:
i think we configured both to be relatively light less than 1 min run
somehow i though that smallEggModel is multiphase but it's not
let's keep smallEggModel as smoke and smallEggModel will be benchmark?

Reply:
Hi there! Correct me if I'm wrong, but what aren't those cases supposed to be referenced by some `ats` cases to be run as integrated tests?

Reply:
yes, `validationCase` should be added in https://github.com/GEOS-DEV/integratedTests/blob/develop/tests/allTests/poroElasticCoupling/poroElasticCoupling.ats

Reply:
OK; could you do it so we can merge this PR?
Or do you want to merge as is with the cases untested?

Reply:
I will prepare ats file modification asap

Reply:
> OK; could you do it so we can merge this PR? Or do you want to merge as is with the cases untested?

Created draft integratedTests PR with new tests here https://github.com/GEOS-DEV/integratedTests/pull/75
Will need help to rebaseline all the tests (all solver tests will technically fail because of the new parameter in NonlinearSolverParameters)